discovery/device/src/Microsoft.Azure.Devices.Discovery.Client.csproj

```diff
@@ -28,7 +28,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <Version>1.0.2</Version>
+    <Version>1.1.0</Version>
```

Comment:
If possible Lets ensure that all the 4 NuGet have the same consistent version since they are all built together.